test/functional/config.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+    browsers: [
+        {
+            platform:    'Windows 10',
+            browserName: 'internet explorer',
+            version:     '11.0',
+            alias:       'ie'
+        },
+        {
+            platform:    'Windows 10',
+            browserName: 'firefox',
+            alias:       'ff'
+        },
+        {
+            platform:    'Windows 10',
+            browserName: 'chrome',
+            alias:       'chrome'
+        }
+    ],
+    testCafe: {
+        hostname: '127.0.0.1',
+        port1:    1335,

```

Comment:
Not all of ports are available on some of saucelabs wms (for example on ios). 
Let's use `2000, 2001, 3000, 3001` ports here.